From 73110ad60326043a7ca275089621fd062c91c9bf Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Fri, 24 Jun 2016 15:20:29 +0200
Subject: [PATCH] Build without bundled Module::Package modules
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This Module-Install-Contributors distribution bundles
inc/Module/Package/Dist/RDF.pm that is loaded from Makefile.PL by
inc::Module::Install via Module::Package via Module::Package::RDF.
Module::Package::RDF run-requires Module::Install::Copyright that
build-requires this Module-Install-Contributors distribution.

This patch breaks the build cycle.

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 Makefile.PL | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile.PL b/Makefile.PL
index d47c2bc..5edcf28 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -1,4 +1,5 @@
-use inc::Module::Package 'RDF:tobyink 0.012';
+use lib 'lib';
+use inc::Module::Package;
 
 contributors "Alan Smithee", "Nicolas Bourbaki";
 contributors "Spartacus";
-- 
2.5.5

